### PR TITLE
Add support for cronjob extractor application nodes

### PR DESCRIPTION
**Pull request by disrupted**

### CODE REVIEW

**Review by philipp94831 — CHANGES_REQUESTED** [review]
What about a test?

**Comment by disrupted** [comment]
> What about a test?

it's included in the extractor plugin

**Comment by philipp94831** [comment]
> > What about a test?
> 
> it's included in the extractor plugin

But there should be one here as well

**Comment by disrupted** [comment]
> But there should be one here as well

hm I think it's already covered. What do you think I should test here?

**Comment by philipp94831** [comment]
> > But there should be one here as well
> 
> hm I think it's already covered. What do you think I should test here?

That an application is created from a cronjob

File: backend/streams_explorer/core/extractor/extractor_container.py [hunk 1]
```diff
@@ -42,9 +42,12 @@ def on_connector_info_parsing(
                 return connector
         return None
 
-    def on_cron_job(self, cron_job: V1beta1CronJob):
+    def on_cron_job(self, cron_job: V1beta1CronJob) -> Optional[V1Deployment]:
```
Comment (philipp94831) [inline backend/streams_explorer/core/extractor/extractor_container.py:45]: Shouldn't this be K8sApp instead of V1Deployment?

Comment (disrupted) [reply backend/streams_explorer/core/extractor/extractor_container.py:45]: WIP, let me mark this as a Draft again

**Review by philipp94831 — COMMENTED** [review]
Are you able to do the thing with the active edges now by checking for K8sAppCronJob and marking all incoming edges just as the outgoing edge?

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**